pkg/cmd/clustertask/start.go

```diff
@@ -48,8 +48,9 @@ import (
 )
 
 var (
-	errNoClusterTask      = errors.New("missing ClusterTask name")
-	errInvalidClusterTask = "ClusterTask name %s does not exist"
+	errNoClusterTask             = errors.New("missing ClusterTask name")
+	errInvalidClusterTask        = "ClusterTask name %s does not exist"
+	errClusterTaskAlreadyPresent = "ClusterTask %s already exists"
```

Comment:
Could you please change it to 
`errClusterTaskAlreadyPresent = "ClusterTask  with name %s already exists"`